apps/trading-e2e/cypress.json

```
@@ -14,6 +14,7 @@
   "chromeWebSecurity": false,
   "projectId": "et4snf",
   "env": {
+    "ETHEREUM_PROVIDER_URL": "https://ropsten.infura.io/v3/4f846e79e13f44d1b51bbd7ed9edefb8",
```

chore (blocking): inconsistent casing

```suggestion
    "ethereumProviderUrl": "https://ropsten.infura.io/v3/4f846e79e13f44d1b51bbd7ed9edefb8",
```